[IMP] l10n_id: 11/12 DPP Adjustment (#12732)
odoo/documentation: content/applications/finance/fiscal_localizations/indonesia.rst

Indonesia
=========

.. _localization_indonesia/DPP:

> 12% VAT & 11/12 DPP Adjustment

Reviewer: ```suggestion
12% VAT & 11/12 DPP adjustment
```

==============================

Indonesia's VAT rate increased to 12% from 11% as of January 1, 2025. To mitigate the impact of the
VAT increase on non-luxury goods and services, *DPP nilai lain* (:dfn:`tax base of other value`) is
applied to those goods and services, set at 11/12 of the untaxed base.

> 

Reviewer: Can you add a custom anchor for this heading? Thanks!

Tax settings
------------

Adjusted DPP computation applies to invoice lines with a tax that falls under the tax group
`Non-luxury Good Taxes (ID)`, and display :guilabel:`DPP Other Value (11 / 12)` on the
:guilabel:`Customer Invoices` and their printouts. To configure, go to

> :menuselection:`Accounting --> Configuration --> Taxes`, and click the tax that applies 11/12 base
> computation. Under :guilabel:`Advanced Options` tab, choose `Non-luxury Good Taxes (ID)` for the
> field :guilabel:`Tax Group`.

Reviewer: ```suggestion
:menuselection:`Accounting --> Configuration --> Taxes`, and select the tax that uses the 11/12 base
computation. Then, under the :guilabel:`Advanced Options` tab, select :guilabel:`Non-luxury Good Taxes (ID)` in the
:guilabel:`Tax Group` field.
```
